.github/workflows/kubernetes-deploy.yaml

```diff
@@ -31,11 +31,27 @@ jobs:
           docker build -t ray:test --build-arg TARGETARCH="amd64" -f ./Dockerfile-ray-node .
           kind load docker-image ray:test
           docker image rm ray:test
-      - name: Build and load proxy
+      - name: Build and load function
         run: |
-          docker build -t proxy:test --build-arg TARGETARCH="amd64" -f ./proxy/Dockerfile .
-          kind load docker-image proxy:test
-          docker image rm proxy:test
+          docker build -t test_function:latest --build-arg TARGETARCH="amd64" -f ./tests/basic/function/Sample-Docker ./tests/basic
+          docker images
+          docker rmi moby/buildkit:latest
```

Comment:
Seeing that this configuration it's a bit weird right now, Aki, and just to not postpone too much this PR how would you see to remove by now the test from `kubernetes-deploy` and do wat @psschwei proposed one moment ago in another PR?